src/components/Calculator.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Button from './Button';
+import Result from './Result';
+import '../App.css';
+
+function Calculator() {
+  const regularButton = 'box-border border-2 border-neutral-400 col-span-1 bg-indigo-500 flex justify-center items-center';
+  const wideButton = 'box-border border-2 border-neutral-400 col-span-2 bg-indigo-500 flex justify-center items-center';
+  const display = 'box-border border-2 border-neutral-400 col-span-4 bg-indigo-500 flex justify-center items-center text-white';
+  const actionColor = 'bg-orange-500';
+  const numberColor = 'bg-stone-200';
+  const displayColor = 'bg-neutral-800';
+  const calculatorStyle = 'grid grid-cols-4 w-80 h-80 box-border border-4 border-neutral-400';
+
+  return (
+    <div className={calculatorStyle}>
+      <Result className={`${display} ${displayColor}`} result="Calculator" />
+
+      <Button className={`${regularButton} ${numberColor}`} identifier="AC" />
+      <Button className={`${regularButton} ${numberColor}`} identifier="+/-" />
+      <Button className={`${regularButton} ${numberColor}`} identifier="%" />
+      <Button className={`${regularButton} ${actionColor}`} identifier="/" />
+
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Calculator.js%20file.-,In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A,-Implement%20the%20design), Please replace the divide symbol you added `/` with this symbol `÷` instead to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/218836370-5286a6b0-3103-4941-aea2-12914937bb0e.png)  | ![image](https://user-images.githubusercontent.com/68796232/218883760-02e32afc-a69b-45a9-9a2e-2169c172decf.png) |